apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/model/EndringBrevRequest.kt

```diff
@@ -0,0 +1,43 @@
+package no.nav.etterlatte.brev.model
+
+import no.nav.etterlatte.brev.behandling.Avdoed
+import no.nav.etterlatte.brev.behandling.Behandling
+import no.nav.etterlatte.brev.behandling.Soeker
+import no.nav.etterlatte.brev.behandling.Utbetalingsinfo
+import no.nav.etterlatte.libs.common.behandling.RevurderingAarsak
+
+class EndringBrevRequest(
+    val saksnummer: String,
+    val utbetalingsinfo: Utbetalingsinfo,
+    val barn: Soeker,
+    val revurderingsaarsak: RevurderingAarsak,
+    val avdoed: Avdoed,
+    override val spraak: Spraak,
+    override val avsender: Avsender,
+    override val mottaker: BrevMottaker,
+    override val attestant: Attestant?
+) : BrevRequest() {
+
+    override fun templateName(): String = "endring"
+
+    companion object {
+        fun fraVedtak(
+            behandling: Behandling,
+            avsender: Avsender,
+            mottaker: BrevMottaker,
+            attestant: Attestant?
+        ): EndringBrevRequest = EndringBrevRequest(
+            spraak = behandling.spraak,
+            avsender = avsender,
+            mottaker = mottaker,
+            attestant = attestant,
+            saksnummer = behandling.sakId.toString(),
+            utbetalingsinfo = behandling.utbetalingsinfo!!,
```

Comment:
Vil vi alltid ha utbetalingsinfo her? Kan sjå for meg at vi kjem til å revurdere ein avvist søknad og så avvise også i revurderinga?

Reply:
Kanskje -- dette er kopiert fra innvilgelse, men som du sier kan ting være annerledes for revurderinger. Enn så lenge har vi bare endringsbrev (vedtaktype == endring) med utbetalingsinformasjon, noe som kanskje holder i framtiden også. Velger å ikke trikse med det nå i alle fall :p